tests/bootstrap.php

```diff
@@ -88,7 +88,7 @@
 
 // Ensure default test connection is defined
 if (!getenv('db_dsn')) {
-    putenv('db_dsn=sqlite://' . TMP . 'debug_kit_test.sqlite');
+    putenv('db_dsn=sqlite:///' . TMP . 'debug_kit_test.sqlite');
```

Comment:
In fact, putenv('db_dsn=sqlite:/' . TMP . 'debug_kit_test.sqlite'); would have worked too.
So it works with 1 or 3 slashes, just not with 2